config/prow/cluster/cherrypick_deployment.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cherrypick
+  namespace: prow
+spec:
+  selector:
+    app: prow
+    component: cherrypick
+  ports:
+  - port: 80
+    targetPort: 8888
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cherrypick
+  labels:
+    app: prow
+    component: cherrypick
+  namespace: prow
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+    app: prow
```

Comment:
Problem is here.. need to fix the indentation for these items under matchLabels.. add 2 spaces and make these part of the matchLabels instead of selector..

Reply:
I fixed the indentation and also changed the value passed to `--github-token-path` flag from `/etc/github/oauth` to `/etc/github/token`. The `cherrypick pod` is Up and Running and below is the text from the pod log:
```
{"client":"github","file":"prow/github/client.go:705","func":"k8s.io/test-infra/prow/github.(*client).log","level":"info","msg":"User()","time":"2021-03-18T10:48:56Z"}
{"client":"github","file":"prow/github/client.go:705","func":"k8s.io/test-infra/prow/github.(*client).log","level":"info","msg":"User()","time":"2021-03-18T10:48:56Z"}
{"client":"github","file":"prow/github/client.go:705","func":"k8s.io/test-infra/prow/github.(*client).log","level":"info","msg":"GetRepos(ltccci, true)","time":"2021-03-18T10:48:56Z"}
{"client":"github","duration":"67.557213ms","file":"prow/github/client.go:707","func":"k8s.io/test-infra/prow/github.(*client).log.func2","level":"debug","msg":"GetRepos(ltccci, true) finished","time":"2021-03-18T10:48:56Z"}
```
@mkumatag Kindly confirm if the above message from the log means the cherrypicker Endpoint is working as expected. Please let me know if anything else needs to be verified.

Reply:
/lgtm